lib/widgets/app.dart

```diff
@@ -112,25 +142,24 @@ class ZulipApp extends StatelessWidget {
       tooltipTheme: const TooltipThemeData(preferBelow: false),
     );
 
-    return GlobalStoreWidget(
-      child: Builder(builder: (context) {
-        final globalStore = GlobalStoreWidget.of(context);
-        // TODO(#524) choose initial account as last one used
-        final initialAccountId = globalStore.accounts.firstOrNull?.id;
-        return MaterialApp(
+    return GlobalStoreWidget(child: Builder(builder: (context) {
+      final globalStore = GlobalStoreWidget.of(context);
+      // TODO(#524) choose initial account as last one used
+      final initialAccountId = globalStore.accounts.firstOrNull?.id;
+      return MaterialApp(
           title: 'Zulip',
           localizationsDelegates: ZulipLocalizations.localizationsDelegates,
           supportedLocales: ZulipLocalizations.supportedLocales,
           theme: theme,
-
           navigatorKey: navigatorKey,
           navigatorObservers: navigatorObservers ?? const [],
           builder: (BuildContext context, Widget? child) {
             if (!ready.value) {
-              SchedulerBinding.instance.addPostFrameCallback(
-                (_) => _declareReady());
+              SchedulerBinding.instance
+                  .addPostFrameCallback((_) => _declareReady());
```

Comment:
Avoid changing unrelated code. See Zulip's commit style guide:
https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html

If your editor made this change automatically, I'd be interested in hearing what editor you use. Then if you can find how to configure it to not do that, I'd be glad for a PR to make that change.

Reply:
@gnprice  Sir
I am using vscode
But I think that was a mistake from my side

Reply:
I see. We do have that configuration for VS Code:
```
$ cat .vscode/settings.json 
// Visual Studio Code settings for the zulip-flutter repo.
{
  // VS Code formats files on save by default.
  // Like Flutter itself, we don't use the Dart autoformatter,
  // so disable that VS Code behavior.
  "[dart]": {
    // This appears redundant with the all-language setting,
    // but it's not: if the user's personal config has a Dart-specific
    // setting, then we need a Dart-specific setting to override it.
    "editor.formatOnSave": false,
    "editor.formatOnType": false,
    "editor.formatOnPaste": false,
  },

  // For that matter, don't go reformatting whole files in any language.
  "editor.formatOnSave": false,

  // This much more focused automatic fix is helpful, though.
  "files.trimTrailingWhitespace": true,
}
```

so yeah, I think it would not have done this automatically.

In any case you'll need to remove this change from your PR. For that, `git restore -p` will be helpful — see `git help restore`.

Reply:
I had used this on my branch
`git restore -p`
##  Output
`No changes.`
What else do you suggest so that I could revert the changes if made

Reply:
For getting help with tools, Zulip itself is a better medium than GitHub — you'll get faster answers that way, because more people will see the question. See this doc:
https://zulip.com/development-community/

For questions about Git in particular, the `#git help` stream is best.

(You'll need to pass additional arguments to `git restore` in addition to the `-p` flag. See `git help restore`, as I mentioned; if you have questions after looking at that, then ask in `#git help` and we can help you out there.)